resources/public/api/conf/1.0/docs/overview.md

```diff
@@ -1,12 +1,18 @@
-The Integration Framework Test Support API allows you to create test data for the following APIs:
-
-- Individuals Employments
-- Individuals Income
-- Individuals Benefits and Credits
-- Individuals Details
-- Organisations Matching
-- Organisations Details
-
-This API does not accept duplicate entries. You must create a unique user using the Create a
-Test User API before submitting data to the employments, income, benefits and credits, 
-and details APIs.
+The Integration Framework (IF) Test Support API allows you to use custom data to test the following APIs:
+
+- Individuals Employments (V2)
+- Individuals Income (V2)
+- Individuals Benefits and Credits (V1)
+- Individuals Details (V1)
+- Organisations Matching (V1)
+- Organisations Details (V1)
+
+External Test is a non-integrated environment. This API allows you to stub IF JSON response objects.
+
+The endpoints in the IF Test Support API mirror the endpoints in IF that the Individuals and Organisations APIs get their data from.
+
+Before calling the endpoints, you will need to create a test user using the [Create Test User API](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/api-platform-test-user/1.0#_create-a-test-user-which-is-an-individual_post_accordion).
```

Comment:
this points specifically to creating an individual, rather than the page in general- would the start of the page be preferable ? ie 
-- Before calling the endpoints, you will need to create a test user using the [Create Test User API](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/api-platform-test-user/1.0#_create-a-test-user-which-is-an-individual_post_accordion).
++ Before calling the endpoints, you will need to create a test user using the [Create Test User API](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/api-platform-test-user).